### PR TITLE
Add request_id and task_id support in cloud functions

### PR DESCRIPTION
**Pull request by aarontp**
Most things are working except for querying by "open" tasks.  Until that's fixed this will leave the default to query for tasks that have been updated in the last day.

### CODE REVIEW

File: tools/gcf_init/index.js [hunk 1]
```diff
@@ -22,38 +22,77 @@ const datastore = Datastore();
...
+ * gcloud beta functions call gettasks \
+ *     --data '{"instance": "turbinia-prod", "kind":"TurbiniaTask",
+ *              "task_id":"abcd1234"}'
  *
  * @param {object} req Cloud Function request context.
  * @param {object} req.body The request body.
- * @param {string} req.body.kind The Datastore kind of the data to retrieve, e.g. "TurbiniaTask".
- * @param {string} req.body.id Id of task to retrieve
+ * @param {string} req.body.kind The kind of Datastore Entity to request
+ * @param {string} req.body.start A date string in ISO 8601 format for the start
```
Comment (berggren) [inline tools/gcf_init/index.js]: I don;t know where this is set, but I would prefer the name be something like "start_time" or similar to be a bit more descriptive.

File: turbiniactl [hunk 2]
```diff
@@ -44,23 +44,42 @@ log = logging.getLogger('turbinia')
 logger.setup()
 
 
-def PrintTaskStatus(project, region, days=1, all_fields=False):
+def PrintTaskStatus(instance, project, region, days=0, task_id=None,
+                    request_id=None, all_fields=False):
   """Prints the recent history for Turbinia Tasks.
 
   Args:
+    instance (string): The Turbinia instance name.
```
Comment (berggren) [inline turbiniactl]: Maybe mention that this is the Pubsub queue name

File: turbiniactl [hunk 3]
```diff
@@ -44,23 +44,42 @@ log = logging.getLogger('turbinia')
...
+  elif task_id:
+    func_args.update({'task_id': task_id})
+  elif request_id:
+    func_args.update({'request_id': request_id})
+
+  response = function.ExecuteFunction('gettasks', func_args)
   if not response.has_key('result'):
-    log.error('Return from GCF does not contain a "result" key.')
+    log.error('No results found')
+    print '\nNo results found.\n'
```
Comment (berggren) [inline turbiniactl]: We should move this to a common function that handles stdout and also logging etc. But fine to do this later.

File: turbiniactl (hunk 2 above)
Comment (aarontp) [reply turbiniactl]: Done.

File: tools/gcf_init/index.js (hunk 1 above)
Comment (aarontp) [reply tools/gcf_init/index.js]: Done.

File: turbiniactl (hunk 3 above)
Comment (aarontp) [reply turbiniactl]: Added #104 for this.

**Pull request merged**